lib/phoenix_live_view/test/live_view_test.ex

```diff
@@ -115,6 +115,16 @@ defmodule Phoenix.LiveViewTest do
 
   alias Phoenix.LiveViewTest.{View, ClientProxy, DOM}
 
+  defmodule LayoutView do
+    @doc false
+
+    use Phoenix.View, root: ""
+
+    def render("live_layout.html", assigns) do
+      render(assigns.view_module, assigns.view_template, assigns)
```

Comment:
Maybe you should do something like:

```suggestion
      ["LAYOUT", render(assigns.view_module, assigns.view_template, assigns)]
```

or similar so we can check the layout was really rendered.